census/vanilla-tf/trainer/task.py

```diff
@@ -44,228 +43,149 @@
                'capital_gain', 'capital_loss', 'hours_per_week', 'native_country',
                'income_bracket')
 
-DEFAULTS = [[0], [''], [0], [''], [0], [''], [''], [''], [''], [''],
-                                    [0], [0], [0], [''], ['']]
+CSV_COLUMN_DEFAULTS = [[0.], [''], [0.], [''], [0.], [''], [''], [''], [''], [''],
+                                    [0.], [0.], [0.], [''], ['']]
 
 # Categorical columns with vocab size
-CATEGORICAL_COLS = (('gender', 2), ('race', 5), ('education', 16), ('marital_status', 7),
+HASH_BUCKET_COLS = (('education', 16), ('marital_status', 7),
                     ('relationship', 6), ('workclass', 9),('occupation', 15), ('native_country', 42))
+KEY_COLS = (('gender', ('female', 'male')), ('race', ('Amer-Indian-Eskimo',
```

Comment:
We're demonstrating using explicit keys for hash buckets. We can remove since we demonstrate this with the label column. But this gives us some sort of parity with the tf.learn example.